extensions/2.0/Vendor/GRIFFEL_bim_data/schema/node.GRIFFEL_bim_data.schema.json

```diff
@@ -0,0 +1,28 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema",
+    "title": "GRIFFEL_bim_data Node Extension",
+    "type": "object",
+    "description": "References type and instance properties of the node.",
+    "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+    "properties": {
+        "buffer": {
```

Comment:
Can this external storage be used simultaneously with JSON-stored values? If so, what data source has the most priority?

Reply:
Instance properties override same type properties - added in node.GRIFFEL_bim_data.schema.json and Readme.md.

Nodes can have both embedded and separately stored properties, in that case both are used. Exporters should take care of possible data duplication. Importers should treat embedded values as of higher priority than values, stored in a separate file, as the file may be obsolete or corrupted more likely.
Corresponding paragraph is added in Readme.